client/app/scripts/superdesk-authoring/views/item-association.html

```diff
@@ -1,4 +1,4 @@
 <figure class="item-association" ng-class="{preview: related}">
     <img ng-if="related" ng-src="{{ related.renditions.viewImage.href }}" ng-click="edit(related)">
-    <figcaption contenteditable="true" ng-model="related.description" ng-change="onchange()"></figcaption>
+    <figcaption contenteditable="true" ng-model="related.description_text" ng-change="onchange()"></figcaption>
```

Comment:
why don't use the `description` field ?
if it's in order to keep the original value, I would name the new one `caption` instead.
Because it will be difficult later to understand why `description` and `description_text` can be different.